packages/contentstack-migration/src/commands/cm/stacks/migration.js

```diff
@@ -134,6 +133,10 @@ class MigrationCommand extends Command {
     } else {
       await this.execSingleFile(filePath, mapInstance);
     }
+    const errLogPath = `${process.cwd()}/migration-logs`;
+    if (fs.existsSync(errLogPath) && !isEmpty(`${errLogPath}/error.logs`)) {
```

Comment:
`!isEmpty(`${errLogPath}/error.logs`)` - This condition is never going to be false. What are we checking here?

Reply:
We are checking whether the logs created have any of the error logs

Reply:
Removed